tests/app/organisation/test_rest.py

```diff
@@ -187,6 +188,43 @@ def test_post_create_organisation(admin_request, notify_db_session, crown):
     assert len(organisation) == 1
 
 
+@pytest.mark.parametrize('org_type', ["nhs_central", "nhs_local", "nhs_gp"])
```

Comment:
We should a test that checks that we _don't_ set this branding for other org types. I think the easiest way to do that would be to tack it on to the previous test, which is for a non-NHS org.